index.html

```diff
@@ -206,10 +206,10 @@ <h2>Contact me</h2>
                       <button type="submit" class="btn last-btn">Get It Touch</button>
                     </div>
                 </fieldset>
-                <img src="image-source/Ellipse 5.png" class="ellipse-5 el" alt="ellipse-5">
-                <img src="image-source/Rectangle 7.png" class="rectangle-7 el" alt="rectangle-7">
+                <img src="image-source/Ellipse 5.png" class="ellipse-5 el" alt="">
+                <img src="image-source/Rectangle 7.png" class="rectangle-7 el" alt="">
             </form>
```

Comment:
- [ ] Great job on adding the alt attribute here 🎉 it really makes your page more accessible. In some cases, it's ok to leave it empty and you did that well in some others but in this case, it's necessary to tell the reader that this is a  logo of ..., in case it's not rendered. 🤓